src/pyOpenMS/pyopenms/addons/consensusmap.py

```
@@ -166,13 +178,72 @@ def to_df(self, columns=None):
+                fids = best.column('consensus_feature_unique_id').to_pylist()
+                for i, fid in enumerate(fids):
+                    if fid is not None and fid not in seen:
+                        seen.add(fid)
+                        keep.append(i)
+                best = best.take(keep)
+                # Build lookup: feature_id -> sequence
+                lookup = {}
+                for fid, seq in zip(
+                    best.column('consensus_feature_unique_id').to_pylist(),
+                    best.column('peptidoform').to_pylist(),
+                ):
```

Ruff (0.15.6)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)